.cloud-build/tests/select_notebooks_test.py

```diff
@@ -0,0 +1,11 @@
+import sys
+sys.path.append("..")
+
+from execute_changed_notebooks_helper import (load_results, select_notebook)
+
+bucket:str = "cloud-build-notebooks-presubmit"
+bucket_file: str = "build_results"
+
+accum = load_results(bucket, bucket_file)
+
+print(accum)
```

Comment:
You can use pytest and assertions to make it a proper test.

Reply:
Yes, that was my next todo to use pytest and then add test for the function select_notebook()

Reply:
done